reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -1229,32 +1229,27 @@ strings.
 
 Some features may not be available on all platforms.
 
-For more information about how .NET provides proxy services to PowerShell, see
-[Accessing the Internet Through a Proxy](/dotnet/framework/network-programming/accessing-the-internet-through-a-proxy).
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
+[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)
+Property to determine the proxy configuration.
 
-Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
-by the following environment variables:
+The value of this property is different rules depending on your platform:
 
-- `$env:http_proxy` or `$env:HTTP_PROXY`
+- **For Windows**: Reads proxy configuration from environment variables or, if those are not
+  defined, from the user's proxy settings.
+- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
+  from the system's proxy settings.
+- **For Linux**: Reads proxy configuration from environment variables or, in case those are not
```

Comment:
```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

It just reads better if they are consistent to me. Feel free to ignore

Reply:
fixed